Core.ApplicationServices/GDPR/Write/DataProcessingRegistrationWriteService.cs

```diff
@@ -220,14 +228,15 @@ private Maybe<OperationError> UpdateOversightDates(DataProcessingRegistration dp
 
         private Maybe<OperationError> UpdateOversightOptions(DataProcessingRegistration dpr, Maybe<IEnumerable<Guid>> oversightOptionUuids)
         {
-            return UpdateMultiAssignment
+            return _assignmentUpdateService.UpdateUniqueMultiAssignment
             (
                 "oversight options",
                 dpr,
                 oversightOptionUuids,
+                oversightUuid => _entityResolver.ResolveEntityFromUuid<DataProcessingOversightOption>(oversightUuid),
```

Comment:
mæh --- hvorfor henten entiteten ud? Din TInput kan vel være int?... brug identity resolveren og keep it simple